jccli/jc_api_v1.py

```diff
@@ -63,7 +60,7 @@ def search_users(self, filter={}):
             users = [user.to_dict() for user in api_response.results]
             return users
         except ApiException as error:
-            raise "Exception when calling SystemusersApi->systemusers_list: %s\n" % error
+            raise Exception("Exception when calling SystemusersApi->systemusers_list: %s\n" % error)
```

Comment:
why not raise as an a jccli.errors.JcCliError (or some other subclass). If these are just being raised to the cli handler it may not make much of a functional difference now but if called these from elsewhere being able to catch something more specific than Exception might be helpful.

Reply:
Added new exception `JcApiException`